src/mlpack/methods/kernel_svm/kernel_svm_main.cpp

```diff
@@ -0,0 +1,431 @@
+/**
+ * @file methods/kernel_svm/kernel_svm_main.cpp
```

Comment:
I think we can work on the cli implementation later once the core implementation is complete.